experiment/build/generate_cloudbuild.py

```diff
@@ -45,33 +86,54 @@ def create_cloud_build_spec(buildable_images, docker_registry):
         if 'depends_on' in image:
```

Comment:
I think everything but dispatcher-image and base-image have a dockerfile. So let's add one to them and we won't need to have this condition (on line 42). And we can get rid of the `if 'dockerfile' in image:` in generate_makefile as well.

Reply:
So a Dockerfile for them with a sort of NoOp?